prac_08/silver_service_taxi_test.py

```diff
@@ -1,8 +0,0 @@
-from prac_08.silver_service_taxi import SilverServiceTaxi
-
-taxi = SilverServiceTaxi("Silver Service Taxi", 100, 2)
-taxi.drive(18)
-print(taxi)
-print()
```

Comment:
is this print call 'print()' a formatting thing to add a blank line?.. can you just amend line 5 to 'print(taxi,"\n")'?